controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/ApplicationPackage.java

```diff
@@ -184,4 +189,27 @@ private static String withoutLegacyDir(String name) {
 
     }
 
+    /** Creates a valid application package that will remove all application's deployments */
+    public static ApplicationPackage createEmptyForDeploymentRemoval() {
```

Comment:
Consider simplifying to `deploymentRemoval()`.

Reply:
Done